### ISSUE TITLE
CI builds by GTitHub actions

### ISSUE THREAD

**Issue by w-bruns** [Normaliz/PyNormaliz#107]
The Travis ci builds have now been successfully replaced by GitHub actions. There are 3 builds:

1. Linux with all optional packages
2. Linux without e-antic, but all the other packages
3. MacOS with all optional packages

Note that only the presence of e-antic has an effect on NormalizModule.cpp. Therefore there is no need for further variations. All 3 builds involve Python 2 and 3.

Two details if the Travis builds are still missing: codecov (not really important) and 

```
  - python setup.py sdist
  - sudo $PIP install --no-index --no-deps -v dist/PyNormaliz-*.tar.gz
  - python -c "import PyNormaliz"
```
  
I guess these lines test whether PyNormaliz can be packaged with pip. I assume that the release transfer to pip is not affected. Nevertheless I will try to bring them back.

**Comment by w-bruns** [comment]
The lines  
```
 - python setup.py sdist
  - sudo $PIP install --no-index --no-deps -v dist/PyNormaliz-*.tar.gz
  - python -c "import PyNormaliz"
```
have been inserted now. They work under Linux, but not under MacOS. Reason not yet clear.

**Comment by w-bruns** [comment]
The problem is
`sudo pip install --no-index --no-deps -v dist/PyNormaliz-2.14.tar.gz`
Under Linux it works without complaint, but on Mac we get
`DEPRECATION: Source distribution is being reinstalled despite an installed package having the same name and version as the installed package. pip 21.1 will remove support for this functionality. A possible replacement is use --force-reinstall. You can find discussion regarding this at https://github.com/pypa/pip/issues/8711.`

In other words: we should not do this. But I am tzrying with `--force-reinstal`
